src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -585,24 +633,20 @@ public class GraphQLGtfsSchema {
             // Additional GTFS Flex fields.
             .field(MapFetcher.field("flex_default_travel_time", GraphQLInt))
             .field(MapFetcher.field("flex_default_zone_time", GraphQLInt))
-            .field(MapFetcher.field("mean_duration_factor", GraphQLFloat))
-            .field(MapFetcher.field("mean_duration_offset", GraphQLFloat))
-            .field(MapFetcher.field("safe_duration_factor", GraphQLFloat))
-            .field(MapFetcher.field("safe_duration_offset", GraphQLFloat))
             // FIXME: This will only returns a list with one stop entity (unless there is a referential integrity issue)
             // Should this be modified to be an object, rather than a list?
             .field(newFieldDefinition()
-                    .type(new GraphQLList(locationsType))
+                    .type(new GraphQLList(locationType))
                     .name("location")
                     .dataFetcher(new JDBCFetcher("locations", "location_id"))
                     .build()
             )
             .build();
 
-    public static final GraphQLObjectType patternStopAreaType = newObject().name("patternStopArea")
+    public static final GraphQLObjectType patternLocationGroupStopsType = newObject().name("patternLocationGroupStop")
             .field(MapFetcher.field("id", GraphQLInt))
             .field(MapFetcher.field("pattern_id"))
-            .field(MapFetcher.field("area_id"))
+            .field(MapFetcher.field("location_group_id"))
             .field(MapFetcher.field("drop_off_booking_rule_id"))
             .field(MapFetcher.field("pickup_booking_rule_id"))
```

Comment:
These have duplicates below (lines 659-660), and one of the sets can be removed. Same at lines 621-622 and 630-631.